apps/web/src/pages/EmployeeProfile/components/CareerObjective/CareerObjectiveSection.tsx

```diff
@@ -0,0 +1,613 @@
+    >
+      <ToggleSection.Header
+        Icon={icon.icon}
+        color={icon.color}
+        level="h3"
+        size="h4"
+        toggle={
+          <ToggleForm.LabelledTrigger
+            sectionTitle={intl.formatMessage({
+              defaultMessage: "Your career objective",
+              id: "vvnsjr",
+              description: "Title for the career objective section",
+            })}
+          />
+        }
+        data-h2-font-weight="base(bold)"
+      >
+        {intl.formatMessage({
+          defaultMessage: "Your career objective",
+          id: "vvnsjr",
+          description: "Title for the career objective section",
+        })}
+      </ToggleSection.Header>
+      <p>{subtitle}</p>
+      <ToggleSection.Content>
+        <ToggleSection.InitialContent>
+          {isNull ? (
+            <ToggleForm.NullDisplay />
+          ) : (
+            <Display employeeProfile={employeeProfile} />
+          )}
+        </ToggleSection.InitialContent>
+        <ToggleSection.OpenContent>
+          <FormProvider {...methods}>
+            <form onSubmit={handleSubmit(handleSave)}>
+              <div
+                data-h2-display="base(flex)"
+                data-h2-flex-direction="base(column)"
+                data-h2-gap="base(x1)"
+              >
+                {/* Classification subsection */}
+                <div
+                  data-h2-display="base(flex)"
+                  data-h2-flex-direction="base(column)"
+                  data-h2-gap="base(x.5)"
+                >
+                  <p id={classificationDescriptionId}>
+                    {intl.formatMessage(
+                      employeeProfileMessages.targetClassification,
+                    )}
+                  </p>
+                  <div
+                    data-h2-display="base(flex)"
+                    data-h2-flex-direction="base(column) p-tablet(row)"
+                  >
+                    <div
+                      data-h2-padding="p-tablet(0, x2, 0, 0)"
+                      data-h2-width="base(100%)"
+                    >
+                      <Select
+                        id="classificationGroup"
+                        label={intl.formatMessage(
+                          employeeProfileMessages.classificationGroup,
+                        )}
+                        name="classificationGroup"
+                        nullSelection={intl.formatMessage(
+                          uiMessages.nullSelectionOptionGroup,
+                        )}
+                        options={groupOptions}
+                        disabled={fetching}
+                        aria-describedby={classificationDescriptionId}
+                      />
+                    </div>
+                    {notEmpty(watchClassificationGroup) && (
+                      <div style={{ width: "100%" }}>
+                        <Select
+                          id="classificationLevel"
+                          label={intl.formatMessage(
```

Comment:
I'm not sure this conditional rendering is working as intended


https://github.com/user-attachments/assets/40fbc2d0-40cf-41ac-8c47-6c75e4c36b16

Reply:
Good catch!  That was a weird one.
[fix classification level rendering](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12844/commits/8f7d956b8ac8096e52b91ec1f3e3da602532272f)